examples/md-flexible/scripts/extractClusterGraphCombine.py

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/python3
+
```

Comment:
please document your scripts in the README in this folder: How to use them and the minimal python version that is needed to run (no need to determine the exact, just as low as you know)